feat: add feed

KoRVeRT opened the pull request.

KoRVeRT committed:
feat: add feed

tonasd commented on src/main/resources/schema.sql:
```diff
@@ -92,3 +92,13 @@ CREATE TABLE IF NOT EXISTS PUBLIC.REVIEWS_USERS
+	TIME_FEED     TIMESTAMP,
+	EVENT_TYPE 	  VARCHAR(16) NOT NULL,
+	OPERATION     VARCHAR(16) NOT NULL,
+	USER_ID       INT REFERENCES PUBLIC.USERS (USER_ID) ON DELETE CASCADE,
```
У нас тип  BIGINT у  PUBLIC.USERS (USER_ID). А здесь INT - несостыковка.

tonasd commented on src/main/java/ru/yandex/practicum/filmorate/model/Event.java:
```diff
@@ -0,0 +1,29 @@
+@ToString
+@EqualsAndHashCode
+@NoArgsConstructor
+@AllArgsConstructor
```
Не увидел испльзования этого конструктора в коде, если это и правда так, то эта строка кажется избыточной

tonasd commented on src/main/java/ru/yandex/practicum/filmorate/service/ReviewService.java:
```diff
@@ -11,17 +14,39 @@
     }
 
     public void removeReview(long reviewId) {
+        Review removeReview = getReviewById(reviewId);
```
removedReview кажется более логичным или reviewToRemove

LeonovichMariia commented on src/main/java/ru/yandex/practicum/filmorate/controller/UserController.java:
```diff
@@ -71,6 +74,12 @@ public List<User> getCommonFriends(@PathVariable Long id, @PathVariable Long oth
+    public List<Event> getUserEvents(@PathVariable int id) {
+        log.info("Event feed for user with id={} received", id);
+        return feedService.getByUserId(id);
+    }
```
Думаю, лучше выводить лог об успешной операции после самой операции)
В остальном все супер!

KoRVeRT committed:
refactor : Change variable name removeReview -> removedReview in removeReview method of ReviewService class,
Merge imports in Event class.
Change logging in getUserEvents method in UserController class

style: final for List<Event> userEvents in method getUserEvents in class UserController

tonasd reviewed (approved):
Всё ок
